book/configuration.rst

```diff
@@ -122,13 +122,13 @@ FrameworkBundle configuration:
 
 .. code-block:: bash
 
-    $ app/console config:dump-reference FrameworkBundle
+    $ bin/console config:dump-reference FrameworkBundle
```

Comment:
Yeah, updated it already (these commands currently didn't work for Windows users).